src/dune/dune_file.ml

```diff
@@ -1970,6 +1971,7 @@ module Coq = struct
        and+ public = field_o "public_name" (Public_lib.decode ())
        and+ synopsis = field_o "synopsis" string
        and+ flags = Ordered_set_lang.Unexpanded.field "flags"
+       and+ boot = field_b "boot"
```

Comment:
This should be guarded by the dune language: `~check:(Dune_lang.Syntax.since Stanza.syntax (2, 3))`

Reply:
Thanks, done!